crates/bin/pindexer/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "pindexer"
+version = {workspace = true}
+authors = {workspace = true}
+edition = {workspace = true}
+repository = {workspace = true}
+homepage = {workspace = true}
+license = {workspace = true}
+publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+cometindex = { path = "../../util/cometindex" } # why doesn't this work with workspace = 
```

Comment:
this crate needs to be added to `[workspace.dependencies]` for `workspace = true` to work here.

Reply:
thanks! til

Reply:
e6cd43891 fixes this. :hammer: